exampleSite/content/fragments/content/content-sidebar-right.md

```diff
@@ -12,6 +12,7 @@ subtitle = "Split in two!"
 [sidebar]
   title = "Sidebar"
   align = "right"
+  sticky = false
```

Comment:
Couldn't sticky be "false" by default and we only need to set `sticky = true` if needed?

Reply:
It is false by default. I just added it in the examples to show off it's existence. I'll replace it with `#sticky = true # Default is false` to make it more clear.